From 5b9d4ef6f8fa43c5502052c4925f948e6154508d Mon Sep 17 00:00:00 2001
From: Jay Yang <jay.yang@intel.com>
Date: Fri, 29 Mar 2024 14:21:10 +0800
Subject: [PATCH] Add ACM new DIDs

New device IDs for ACM.

[Internal]
Commit_Type: Feature
Platforms: Xe_Hpm
OS: Linux
Feature impact: N/A
Resolves: VSMGWL-72681
Related-to: VSMGWL-72681
Klocwork: PASS
TP_Passed: N/A
IP Scan: PASS
---
 media_driver/linux/Xe_M/ddi/media_sysinfo_xe.cpp | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/media_driver/linux/Xe_M/ddi/media_sysinfo_xe.cpp b/media_driver/linux/Xe_M/ddi/media_sysinfo_xe.cpp
index 7ff3527f7..58f96e88c 100644
--- a/media_driver/linux/Xe_M/ddi/media_sysinfo_xe.cpp
+++ b/media_driver/linux/Xe_M/ddi/media_sysinfo_xe.cpp
@@ -482,3 +482,10 @@ static bool dg2Gt4Device56C0 = DeviceInfoFactory<GfxDeviceInfo>::
 
 static bool dg2Gt4Device56C1 = DeviceInfoFactory<GfxDeviceInfo>::
     RegisterDevice(0x56C1, &dg2Gt4Info);
+
+static bool dg2Gt4Device56BE = DeviceInfoFactory<GfxDeviceInfo>::
+    RegisterDevice(0x56BE, &dg2Gt4Info);
+
+static bool dg2Gt4Device56BF = DeviceInfoFactory<GfxDeviceInfo>::
+    RegisterDevice(0x56BF, &dg2Gt4Info);
+
-- 
2.34.1

